source/project-stories/reed-learning.erb

```diff
@@ -0,0 +1,156 @@
+  image: "/project-stories/reed-learning/learners-feedback.jpg"
+  image_description: "Reed Learning feedback from learners"
+
+team:
+  title:
+  people:
+    - full_name: "Leon Odey Knight"
+      image: "/project-stories/reed-learning/team/leon.png"
+      link: "/people/leon-odey-knight"
+
+    - full_name: "Tyler Savin"
+      image: "/project-stories/reed-learning/team/tyler.png"
+      link: "/people/tyler-savin"
+
+    - full_name: "Neil van Beinum"
+      image: "/project-stories/reed-learning/team/neil.png"
+      link: "/people/neil-van-beinum"
+
+    - full_name: "Henry Turner"
+      image: "/project-stories/reed-learning/team/henry.png"
+      link: "/people/henry-turner"
+
+    - full_name: "Martyn Evans"
+      image: "/project-stories/reed-learning/team/martyn.png"
+      link: "/people/martyn-evans"
+
+site:
+  link: ""
+  text: "Visit the site and check it out for yourself"
+
+links:
+  - title: "Lorem ipsum"
+    source: "Lorem ipsum"
+    type: "news"
+    link: ""
+
+  - title: "Lorem ipsum"
+    source: "Lorem ipsum"
+    type: "news"
+    link: ""
+
+related:
+  title: "Related"
+  articles:
+    - title: "Lorem ipsum"
+      type: "news"
+      link: ""
+
+    - title: "Lorem ipsum"
+      type: "blog"
+      link: ""
+
+    - title: "Lorem ipsum"
+      type: "project story"
+      link: ""
+
+
+---
+
+<%= partial 'project-stories/page_header' %>
+
+<div class="container" data-header-waypoint>
+  <%= partial 'project-stories/full_image', locals: { section: current_page.data.header } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.introduction } %>
+
+  <section class="highlight--reed-learning">
+    <div class="highlight__tile--two">
+      <p class="highlight__large-text"><%= current_page.data.statistics_1.number %><p>
+      <p class="highlight__small-text"><%=current_page.data.statistics_1.text %></p>
+      <p class="highlight__large-text"><%= current_page.data.statistics_2.number %><p>
+      <p class="highlight__small-text"><%=current_page.data.statistics_2.text %></p>
+    </div>
+  </section>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.innovating_product } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.cultural_change_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.understanding_customer } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.getting_confidence_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.new_customers } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.team_learning_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.innovation_strategy } %>
+
+  <%= partial 'project-stories/project_team' %>
```

Comment:
<img width="983" alt="screen shot 2015-12-15 at 10 24 25" src="https://cloud.githubusercontent.com/assets/885223/11808074/10ff6b48-a316-11e5-95c2-c0003b8387e6.png">

Two things here, it's missing the title and also wondering if we could align the caricatures so that the eyes are inline? Here's a few other project stories for reference:

<img width="980" alt="screen shot 2015-12-15 at 10 28 59" src="https://cloud.githubusercontent.com/assets/885223/11808181/bd5da864-a316-11e5-8627-dbaa2ea3186c.png">
<img width="993" alt="screen shot 2015-12-15 at 10 29 07" src="https://cloud.githubusercontent.com/assets/885223/11808180/bd5c6684-a316-11e5-950a-288b6a5545ac.png">
<img width="976" alt="screen shot 2015-12-15 at 10 29 13" src="https://cloud.githubusercontent.com/assets/885223/11808182/bd619596-a316-11e5-8d4f-5882286f5738.png">

It's much easier to do this to the images directly instead of individual CSS styles. What I tend to do is put all the images onto one canvas and position each person so the middle of the eyes match up, and also trying to resize the images so that the forehead and chin and at the same positions (doesn't always work that easy though).